src/pages/docs/deployments/kubernetes/kustomize/index.md

```diff
@@ -0,0 +1,55 @@
+   Everything else is defined in the `kustomization.yaml` files directly, and overlays should match the same environment structure defined in Octopus itself. 
+
+2. **No overlays**  
+   In this scenario, you may define two overlays, one being for local use outside Octopus, so you can test your `yaml` files. The other overlay is used exclusively by Octopus.
+   In the Octopus overlay, you add [variable substitution syntax](/docs/projects/variables/variable-substitutions) directly into `kustomization.yaml` and any other `yaml` files in the overlay folder, as well as `.env` files.
+   This scenario is better suited for customers with many environments and/or tenants, where customisations are needed for each target.
+
+3. **A mix of both!**  
+   This would be the ideal scenario when you have a small number of environments and many tenants.
+   In this scenario, you model your overlays based on the environments, with the different settings hardcoded in the `kustomization.yaml` file, and you use Octopus [substitution syntax](/docs/projects/variables/variable-substitutions) to define the different tenant properties in `kustomization.yaml` file.
+   Again as in the previous scenarios, we would use `.env` files for secrets via [secretGenerator](https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_secretgenerator_).
+
+*When you use Kustomize to template configurations for many applications*
+
+4. **A mix of both — templates**  
+   You can go beyond configuration for a single application with Octopus and Kustomize. Imagine you have a hundred applications you deploy to Kubernetes. Some of them might have universal traits, like a group of API applications or a group of databases. Therefore, parts of the configuration will be universal for all the apps or a group of apps. There likely be app-unique parameters (like a container image).  
+   You can combine overlays and Octopus variables to create and use one template for all the apps. In this scenario, you would have a set of base files same for all the apps. One or more levels of overlays to add customizations for an app group. You can introduce another layer of overlays for environments. Finally, app-specific parameters (e.g. container images, tags and labels) can be defined in Octopus variables.  
+   In this scenario, tenant or environment-specific parameters can be added to any overlay. For example, you can add prefixes for tenants; also consider using [system variables](https://octopus.com/docs/projects/variables/system-variables).
+
+## Kustomization file directory
+
```

Comment:
A brief introduction of how this field is used might be helpful here. Something like: "during deployment, kustomize reads the kustomization.yaml file located at this path to perform manifest yaml transforms". It will help people link their kustomize knowledge with this step.

Reply:
@YihaoOct feel free to modify this docs and add it